packages/cli/src/lib/decorators/preconditions/git.ts

```diff
@@ -0,0 +1,11 @@
+import {appendCmdIfWindows} from '../../utils/os';
+import {getBinInstalledPrecondition} from './binPreconditionsFactory';
+
+export const IsGitInstalled = getBinInstalledPrecondition(
+  appendCmdIfWindows`git`,
```

Comment:
git is a .exe on Windows, so you do not need `appendCmdIfWindows`
```suggestion
'git',
```
(iirc, based on what we do with Node, I don't think we need to add .exe either)